collectors/googlestackdriver/package-lock.json

```diff
@@ -0,0 +1,4203 @@
+{
```

Comment:
this file is not needed.

Reply:
I still think that its better to include these lock files. We use similar ones in our erlang services. However we do hard pin libraries, so it might be less important.